content/en/profiler/enabling/supported_versions.md

```diff
@@ -46,8 +46,8 @@ To access additional profiling features, use the minimum versions summarized in
 
 |                                   | [Java][1]  | [Python][2]  |  [Go][3]   |  [Ruby][4] |   [Node.js][5]  |  [.NET][6]   |   [PHP][7]  | [Rust/C/C++][8] |
 |-----------------------------------|:-------:|:-------:|:------------:|:------:|:---------:|:-------:|:------:|:----------:|
-| {{< ci-details title="Code Hotspots" >}}Find specific lines of code related to performance issues. <a href="/profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces">Learn more</a>{{< /ci-details >}}   | [JDK&nbsp;8+][17] | tracer&nbsp;0.44.0 | tracer&nbsp;1.37.0 | tracer&nbsp;0.48.0 | beta | tracer&nbsp;2.7.0 | tracer&nbsp;0.71.0 |      |
-| {{< ci-details title="Endpoint Profiling" >}}Identify endpoints that are bottlenecks or responsible for heavy resource consumption. <a href="/profiler/connect_traces_and_profiles/#endpoint-profiling">Learn more</a>{{< /ci-details >}}   | [JDK&nbsp;8+][17] | tracer&nbsp;0.54.0 | tracer&nbsp;1.37.0 | tracer&nbsp;0.52.0 | beta | tracer&nbsp;2.15.0 | tracer&nbsp;0.79.0 |      |
+| {{< ci-details title="Code Hotspots" >}}Find specific lines of code related to performance issues. <a href="/profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces">Learn more</a>{{< /ci-details >}}   | [JDK&nbsp;8+][17] | tracer&nbsp;0.44.0 | tracer&nbsp;1.37.0 | tracer&nbsp;0.48.0 | Node&nbsp;14:&nbsp;tracer&nbsp;3.45.0, Node&nbsp;16:&nbsp;tracer&nbsp;4.24.0, Node&nbsp;18+:&nbsp;tracer&nbsp;5.0.0 | tracer&nbsp;2.7.0 | tracer&nbsp;0.71.0 |      |
```

Comment:
```suggestion
| {{< ci-details title="Code Hotspots" >}}Find specific lines of code related to performance issues. <a href="/profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces">Learn more</a>{{< /ci-details >}}   | [JDK&nbsp;8+][17] | tracer&nbsp;0.44.0 | tracer&nbsp;1.37.0 | tracer&nbsp;0.48.0 | tracer&nbsp;3.45.0,&nbsp;4.24.0,&nbsp;5.0.0 | tracer&nbsp;2.7.0 | tracer&nbsp;0.71.0 |      |
```

Reply:
I think most customers will figure out by themselves that this is related to their node version